samples/managed-identity/Dockerfile

```diff
@@ -1,27 +1,31 @@
 #See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.
 
-FROM mcr.microsoft.com/dotnet/aspnet:7.0 AS base
+FROM mcr.microsoft.com/dotnet/aspnet:7.0-cbl-mariner2.0 AS base
+ARG TARGETARCH
 WORKDIR /app
 EXPOSE 80
 
-FROM mcr.microsoft.com/dotnet/sdk:7.0 AS build
+FROM mcr.microsoft.com/dotnet/sdk:7.0-cbl-mariner2.0 AS build
+ARG TARGETARCH
 WORKDIR /src
 COPY "mi-webapp.csproj" .
 
-RUN dotnet restore "mi-webapp.csproj"
+RUN dotnet restore "mi-webapp.csproj" -a ${TARGETARCH}
 
 COPY . .
 
 WORKDIR "/src/."
-RUN dotnet build "mi-webapp.csproj" -c Release -o /app/build
+RUN dotnet build "mi-webapp.csproj" -c Release -o /app/build -a ${TARGETARCH}
 
 FROM build AS publish
-RUN dotnet publish "mi-webapp.csproj" -c Release -o /app/publish
+RUN dotnet publish "mi-webapp.csproj" -c Release -o /app/publish -a ${TARGETARCH}
 
 FROM base AS final
 WORKDIR /app
 COPY --from=publish /app/publish .
-RUN apt-get update -y
-RUN apt-get upgrade -y
-RUN apt-get install -y libsecret-1-0
+# RUN tdnf clean all
```

Comment:
Can we remove these commented-out lines? Is the `libsecret` dependency no longer required?

Reply:
@peterbom Yes sure will remove them I am was waiting for test results from E2E to make sure it is not used.